core-services/secret-mirroring/_mapping.yaml

```diff
@@ -234,3 +234,9 @@ secrets:
   to:
     namespace: test-credentials
     name: common-templates-container-disk-puller
+- from:
+    namespace: brew-registry-pullsecret
```

Comment:
That namespace doesn't exist

Reply:
@alvaroaleman 
I believe we have created a namespace on the cluster we were given access to. 
Personally, I can see a namespace created with access to admins. 
Are there any other groups I have to give access to? 
@petr-muller @stevekuznetsov

Reply:
This is not about access, I am admin on that cluster, that ns doesn't exist:
```
$ k get secret -n brew-registry-pullsecret registry-pullsecret --context=api.ci
Error from server (NotFound): namespaces "brew-registry-pullsecret" not found
```